spec/ddtrace/contrib/active_support/notifications/subscriber_spec.rb

```diff
@@ -44,7 +48,7 @@
 
           context 'after #subscribe! has been called' do
             before(:each) do
-              test_class.send(:on_subscribe, &proc {})
+              test_class.send(:on_subscribe, &proc { subscribe(:event, :span) {} })
```

Comment:
See feedback from my other comment; this probably has to change.

Reply:
since I reverted the change, I've switched to using `instance_eval` directly here.